src/components/SeoForm/SeoForm.tsx

```diff
@@ -28,28 +25,7 @@ const maxTitleLength = 70;
 const maxDescriptionLength = 300;
 
 const useStyles = makeStyles(
```

Comment:
question: couldn't we get rid totally makeStyle here?

Reply:
I would love to but those styles apply to the `Textarea` component which we don't have yet ([issue](https://github.com/saleor/macaw-ui/issues/325)). It also adds some nested selectors so inline styles of sprinkles will not work here 😞